### PR TITLE
Reduce log spam 

### PR DESCRIPTION
**Pull request by fasaxc**
## Todos
- [ ] Unit tests (full coverage)
- [ ] Integration tests (delete as appropriate) In plan/Not needed/Done
- [ ] Documentation
- [ ] Backport
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: conntrack/conntrack.go [hunk 1]
```diff
@@ -88,7 +89,11 @@ func (c Conntrack) RemoveConntrackFlows(ipVersion uint8, ipAddr net.IP) {
 			if err == nil {
 				logCxt.Debug("Successfully removed conntrack flows.")
 				break
+			} else if bytes.Contains(output, []byte("0 flow entries have been deleted")) {
+				logCxt.Debug("conntrack tool didn't find any flows.")
+				break
```
Comment (neiljerram) [inline conntrack/conntrack.go:94]: How is this addition helping?

File: routetable/route_table.go [hunk 2]
```diff
@@ -825,7 +825,7 @@ func (r *RouteTable) fullResyncRoutesForLink(logCxt *log.Entry, ifaceName string
 		logCxt := logCxt.WithField("dest", dest)
 		// Check if we should remove routes not added by us
 		if !r.removeExternalRoutes && route.Protocol != r.deviceRouteProtocol {
-			logCxt.Info("Syncing routes: not removing route as its not marked as Felix route")
+			logCxt.Debug("Syncing routes: not removing route as its not marked as Felix route")
```
Comment (neiljerram) [inline routetable/route_table.go:828]: Add the missing apostrophe here?  (`its` -> `it's`)

File: conntrack/conntrack.go (hunk 1 above)
Comment (fasaxc) [reply conntrack/conntrack.go:96]: Added a comment; conntrack treats it as a failure if it doesn't find any flows to delete (but for us that's a success!)

Comment (fasaxc) [reply conntrack/conntrack.go]: Yes, I figured that out later :facepalm:

**Pull request merged**